CITATION.cff

```diff
@@ -24,5 +24,10 @@ authors:
     family-names: Zacharias
     affiliation: NSF NCAR
     orcid: 'https://orcid.org/0000-0002-2666-8493'
+  - name: GeoCAT Community Contributors
```

Comment:
I think I'd advocate for explicitly including folks (especially for the time being while it's pretty easy to manage) and decoupling the all contributors stuff from the citation (i.e removing this for now and manually syncing the info).  

That said, I'm curious what the intent was here - is it trying to tie in some metadata about who contributed what?

Reply:
Oh! Maybe I misunderstood the conclusions from our last meeting convo.

So, yes, I had intended that we explicitly include folks that contribute to the level of "authorship". This here is just for also linking to non-authorship contributions (and, I suppose who contributed what metadata for official "authorship" contributors).

Reply:
Would you be opposed to leaving it out for now?  I think it's a little confusing since it's effectively the same people, but infers maybe an authors and others categorization.  This is maybe a me thing though so if you prefer this way I think it's fine.

Reply:
I don't have strong feelings here, we can always add it back later if the people listed in both meaningfully diverges.